### DISCUSSION TITLE
The difference between bot.inventory and bot.registry

### DISCUSSION THREAD

**Discussion by Vyakoo** [PrismarineJS/mineflayer#3114] [Q&A]
I have read the documentation and have not found a specific difference between bot.inventory.items() and bot.registry.itemByName.
Can you explain it to me?

**Comment by amoraschi** [comment]
bot.inventory.items() returns an Array of items in the bot's inventory
bot.registry.itemByName returns an Array of all items

I recommend visiting both [prismarine-windows](https://github.com/PrismarineJS/prismarine-windows) and [minecraft-data](https://github.com/PrismarineJS/minecraft-data) for more information, or joining the Discord server for quicker answers
